recipes/fine/meta.yaml

```diff
@@ -0,0 +1,78 @@
+package:
+  name: fine
+  version: 2.2.2
+
+source:
+  url: https://github.com/FZJ-IEK3-VSA/FINE/archive/refs/tags/v2.2.2.tar.gz
+  sha256: 1e8bff5de0770e1b4e6aba4f9502c107781bb4b9e1faff5d99db32f836cb0fc0
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install --no-deps .
+
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools-git
+  run:
+    - python  >=3.6
+    - libiconv
+    - gdal 
+    - geopandas
+    - jupyter
+    - notebook
+    - libpysal
+    - pip
+    - descartes
+    - glpk
+    - openpyxl
+    - matplotlib-base
+    - xlrd 
+    - pyomo
+    - numpy
+    - pandas
+    - gpyopt
+    - networkx
+    - scipy
+    - pwlf
+    - pydoe
+    - sphinx_rtd_theme
+    - scikit-learn
+    - xarray
+    - statsmodels
+    - seaborn
+    - rasterio
+    - netcdf4
+    - pytest
+    - pytest-cov
+    - pylint
+    - nbval
+    - black  
+    - tsam
+    - wquantiles
+    - glpk
+
+test:
+  imports:
+    - FINE
+    - pandas
+    - numpy
+  source_files:
+    - .
+  requires:
+    - pytest
+  commands:
+    - pytest test/
+
+about:
+  home: https://github.com/FZJ-IEK3-VSA/FINE.git
+  summary: The FINE python package provides a framework for modeling, optimizing and assessing energy systems. With the provided framework, systems with multiple regions, commodities and time steps can be modeled. Target of the optimization is the minimization of the total annual cost while considering technical and enviromental constraints. Besides using the full temporal resolution, an interconnected typical period storage formulation can be applied, that reduces the complexity and computational time of the model.
```

Comment:
This summary is quite long. The GitHub page has:

     The FINE python package provides a framework for modeling, optimizing and assessing energy systems